### ISSUE TITLE
Country select escape key does not work with VoiceOver [AddressComplete]

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#7433]
Using VoiceOver when entering the drop down list, the escape key does nothing. I suspect VoiceOver may be overriding the custom escape key or similar. I think this is not a direct bug with AddressComplete and more related to the type ahead list. Though it should eventually be fixed.

**Comment by thiessenp-cds** [comment]
The bug seems to happen on the initial tab entry into the Country dropdown. Once tabbed in if TAB is keyed to leave the dropdown it will remain active. The user can then tab to other inputs while it remains open.

https://github.com/user-attachments/assets/6d6796ac-5ea2-4d74-b915-721fa513b707
